modules/claims_api/spec/workers/claim_uploader_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'sidekiq/testing'
+Sidekiq::Testing.fake!
+
+RSpec.describe ClaimsApi::ClaimUploader, type: :job do
+  before(:each) do
+    Sidekiq::Worker.clear_all
+  end
+
+  subject { described_class }
+
+  let(:user) { FactoryBot.create(:user, :loa3) }
+  let(:auth_headers) do
+    EVSS::DisabilityCompensationAuthHeaders.new(user).add_headers(EVSS::AuthHeaders.new(user).to_h)
+  end
+
+  let(:claim) do
```

Comment:
you don't actually need this block, you're letting factory bot make the claim in this case by just calling `create(:supporting_document)`

Reply:
Yep you're right, I had it in a different iteration when I was trying to test relationships in the worker, ended up testing in the model instead. Thanks